gulpfile.js

```diff
@@ -42,6 +42,7 @@ embeddedWidgetPath = './app/views/components/widget';
 async function bundleWidgetJS() {
   return browserify("uswds/widget-uswds.js", {
     paths: ['./node_modules'],
+    standalone: 'fbaUswds'
```

Comment:
The standalone option is required for Browserify to create a bundle such that the code inside the bundle can be accessed from code outside the bundle.

https://stackoverflow.com/questions/23296094/browserify-how-to-call-function-bundled-in-a-file-generated-through-browserify